.github/workflows/python.yml

```diff
@@ -0,0 +1,29 @@
+name: Python CI
+
+# trigger
+on:
+  push:
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.10']
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+
+    - name: Prepare python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Run tests
+      run: |
+        cd src/python
+        pip install --upgrade pip
+        pip install -e .
```

Comment:
Ideally install from an sdist and try to load a PEtab problem, to ensure all data files have been added to the package correctly.